docs/rules/expiring-todo-comments.md

```diff
@@ -0,0 +1,108 @@
+# Add conditions to TODO comments to make them error
+
+Makes it possible to pass arguments to TODO and FIXME comments to trigger errors.
+
+This rule also implements [`eslint/no-warning-comments`](https://eslint.org/docs/rules/no-warning-comments) behavior so you should disable that in favor of using this.
+
+For projects with legacy branches, such as long term supported older versions, there might be situations where you fix TODOs in the master branch but on the other branches you leave it missing. Choose carefully which projects you would want this feature.
+
+Supported conditions:
+
+- `[YYYY-MM-DD]` to define a due date.
+- `[>1]` or `[>=2]` to expire at some version (from package.json). No whitespace.
+- `[+package]` or `[-package]` to expire when you add/remove a package.
+- `[package@>1]` or `[package@>=2]` to expire when a package hits some version. No whitespace.
+- `[engines node>8]` or `[engines node>=8]` to expire when package bump supported engines. Don't use spaces between the comparison.
```

Comment:
I think we should make this:

```suggestion
- `[engines:node>8]` or `[engines:node>=8]` to expire when package bump supported engines. Don't use spaces between the comparison.
```

So completely remove whitespace from the conditions.

Reply:
Should whitespaces around `:` be valid?

1. `[engines : node>8]`
2. `[engines: node>8]`
3. `[engines :node>8]`
4. Multiple whitespaces

Reply:
Using it now